imagetest/test_suites/image_validation/license_test.go

```diff
@@ -117,6 +118,7 @@ var licenses = []string{
 	`Open Market permits you to use, copy, modify, distribute, and license this Software and the Documentation for any purpose, provided that existing copyright notices are retained in all copies and that this notice is included verbatim in any distributions. No written agreement, license, or royalty fee is required for any of the authorized uses.`,
 	`This software is made available under the terms of *either* of the licenses found in LICENSE.APACHE or LICENSE.BSD. Contributions to cryptography are made under the terms of *both* these licenses.`,
 	`This library.*is public domain software`,
+	`This Font Software is licensed under the SIL Open Font License`,
```

Comment:
maybe a nitpick, but we usually cover these files with EITHER a license name OR a license text check; you've added both but i think one will suffice. usually we prefer license name, i think